src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private static final double PI = Math.PI;
```

Comment:
I dont think we need to make this value as constant, lets just use `Math.PI` where we need